core/templates/pages/topic-editor-page/services/topic-editor-state.service.ts

```diff
@@ -560,18 +560,26 @@ export class TopicEditorStateService {
    * has been successfully updated.
    */
   updateExistenceOfTopicUrlFragment(
-      topicUrlFragment: string, successCallback: () => void): void {
+      topicUrlFragment: string,
+      successCallback: () => void,
+      errorCallback: () => void
+  ): void {
     this.editableTopicBackendApiService.doesTopicWithUrlFragmentExistAsync(
       topicUrlFragment).then((topicUrlFragmentExists) => {
       this._setTopicWithUrlFragmentExists(topicUrlFragmentExists);
       if (successCallback) {
         successCallback();
       }
-    }, (error) => {
-      this.alertsService.addWarning(
-        error ||
-        'There was an error when checking if the topic url fragment ' +
-        'exists for another topic.');
+    }, (errorResponse) => {
+      if (errorCallback) {
+        errorCallback();
+      }
+      if (errorResponse.status !== 400) {
```

Comment:
Whenever we enter a url fragment in topic editor, backend Api call is used to check whether the url fragment is already exists or not!. This Api call uses http link which is generated using url fragment(that we entered). So, if we enter wrong url fragment (which do not satisfy reg-ex) a wrong http link is generated which causes server to responed with 400 bad request error (because of reg-ex mis-match).
after 400 error is reported errorcallback is used which causes error to display in console and unnecessary pop-up is showed to user. Just to avoid that behaviour i added this check!

Reply:
Perhaps add this explanation as a comment (make it more concise, if possible)

Reply:
Done!